src/main/java/com/amuzil/omegasource/magus/network/MagusNetwork.java

```diff
@@ -1,19 +1,76 @@
 package com.amuzil.omegasource.magus.network;
 
 import com.amuzil.omegasource.magus.Magus;
+import com.amuzil.omegasource.magus.network.packets.client_executed.RegisterModifierListenersPacket;
+import com.amuzil.omegasource.magus.network.packets.client_executed.SkillTriggeredPacket;
+import com.amuzil.omegasource.magus.network.packets.client_executed.UnregisterModifierListenersPacket;
+import com.amuzil.omegasource.magus.network.packets.server_executed.FormActivatedPacket;
+import com.amuzil.omegasource.magus.network.packets.server_executed.SendModifierDataPacket;
+import com.amuzil.omegasource.magus.network.packets.server_executed.StartBendingPacket;
 import net.minecraft.resources.ResourceLocation;
+import net.minecraft.server.level.ServerPlayer;
+import net.minecraftforge.network.NetworkDirection;
 import net.minecraftforge.network.NetworkRegistry;
 import net.minecraftforge.network.simple.SimpleChannel;
 
 public class MagusNetwork {
     private static final String PROTOCOL_VERSION = "1.0.0";
+    private static int packetId = 0;
     public static final SimpleChannel CHANNEL = NetworkRegistry.newSimpleChannel(
             new ResourceLocation(Magus.MOD_ID, "main"),
             () -> PROTOCOL_VERSION,
             PROTOCOL_VERSION::equals,
             PROTOCOL_VERSION::equals
     );
 
+    private static int nextID() {
+        return packetId++;
+    }
+
     public static void registerMessages() {
```

Comment:
We may want a generic `registerMessage()` method for those who wish to use our network but with their own packets. Just a thought.

Reply:
They would just call MagusNetwork.CHANNEL.messageBuilder from their own mod, we cant make it generic because the fromBytes method of a Packet is static, and we cant interface that